integration-test/src/test/java/com/datastrato/gravitino/integration/test/client/AuditIT.java

```diff
@@ -35,17 +37,33 @@ public void testAuditMetalake() throws Exception {
     String metalakeAuditName = RandomNameUtils.genRandomName("metalakeAudit");
     String newName = RandomNameUtils.genRandomName("newmetaname");
 
-    GravitinoMetalake metaLake =
-        client.createMetalake(metalakeAuditName, "metalake A comment", Collections.emptyMap());
+    Awaitility.await().atMost(1, TimeUnit.MINUTES).until(() -> isGravitinoServerUp());
+
+    GravitinoMetalake metaLake = client.createMetalake(metalakeAuditName, "metalake A comment", Collections.emptyMap());
     Assertions.assertEquals(expectUser, metaLake.auditInfo().creator());
     Assertions.assertNull(metaLake.auditInfo().lastModifier());
-    MetalakeChange[] changes =
-        new MetalakeChange[] {
-          MetalakeChange.rename(newName), MetalakeChange.updateComment("new metalake comment")
-        };
+    MetalakeChange[] changes = new MetalakeChange[] {
+        MetalakeChange.rename(newName), MetalakeChange.updateComment("new metalake comment")
+    };
     metaLake = client.alterMetalake(metalakeAuditName, changes);
     Assertions.assertEquals(expectUser, metaLake.auditInfo().creator());
     Assertions.assertEquals(expectUser, metaLake.auditInfo().lastModifier());
     client.dropMetalake(newName);
   }
+
+  // Method to check if the Gravitino server is up
+  private boolean isGravitinoServerUp() {
```

Comment:
Can you move the logic to `AbstractIT` after here https://github.com/datastrato/gravitino/blob/0424c76b6afb2d41550d5a79ad0039b05bc1f3ef/integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java#L212?
Then all tests can be repeated to verify that the Gravitino server is ready.

Reply:
Ok! thank you for your answer.